symfony.lock

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
What is a Symphony.lock for?

Reply:
It's part of Flex, which provides Composer utilities to go along with Symfony. It's kind of cool actually, packages (like PHPUnit) can provide Flex recipes that update files on disk (such as tests/bootstrap.php).

symfony.lock is what Flex uses to keep track of what recipe versions have been applied.